pkg/tsdb/testdata/kinds/dataquery/types_dataquery_gen.go

```diff
@@ -0,0 +1,209 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
```

Comment:
This should end up in pkg/tsdb/testdatasource, but isn't. Seems to (incorrectly) use the plugin name to figure out where to go

Reply:
there's the same issue with `grafanads`, which has different dir names in frontend vs. backend

AIUI the deciding question here is a long-term/big picture one - the big issue is that using dir name, rather than plugin id, is ultimately something that only really makes sense in core. Naming inconsistency is already causing a fair bit drag on the system (see other comment on this PR for a delightful example), so it's an area i'd prefer to keep things simpler.

i see the issue here, though, namely that `testdata` is almost a reserved keyword for go tooling...hmm

i think it probably wouldn't kill us to add a hardcoded special case for this one

Reply:
added a hardcoded special case, this is now fixed

Reply:
Thanks